libs/hdf-converters/package.json

```diff
@@ -59,7 +62,7 @@
     "yaml": "^2.1.1"
   },
   "devDependencies": {
-    "@types/jest": "^27.0.0",
+    "@types/jest": "^29.0.0",
```

Comment:
I believe https://github.com/mitre/heimdall2/pull/5700 details some issues occurring w/ Jest v29. Should play it safe and downgrade this back to v27.